fossunited/fossunited/web_template/foss_footer/foss_footer.html

```diff
@@ -59,29 +59,29 @@
         display: flex;
         flex-direction: column;
         gap: 0.5rem;
-        font-weight: 500;
+        font-weight: var(--fw-medium);
     }
 
     .footer_parent_label p{
         margin: 0;
-        font-size: 0.95rem;
-        color: #EBEBEB;
+        font-size: var(--text-base);
```

Comment:
You may not need to specify the font-size to base everytime, as --text-base size is default for :root

Reply:
ok - fixing it.